interfaces/IBrowser.h

```diff
@@ -151,6 +154,28 @@ namespace Exchange {
         virtual uint32_t CollectGarbage() = 0;
     };
 
+    // @json 1.0.0 @text:legacy_lowercase @uncompliant:extended
+    struct EXTERNAL IWebBrowserExt : virtual public Core::IUnknown {
+
+        enum { ID = ID_WEB_BROWSER_EXT };
+
+        using IStringIterator = RPC::IIteratorType<string, RPC::ID_STRINGITERATOR>;
+
+        // @brief Removes contents of a directory from the persistent storage
+        // @alt:deprecated delete
+        // @description Use this method to recursively delete contents of a directory
+        // @param path: Path to directory (within the persistent storage) to delete contents of (e.g. .cache/wpe/disk-cache)
+        // @retval ERROR_UNKNOWN_KEY The given path cannot be empty
+        virtual Core::hresult DeleteDir(const string& path) = 0;
```

Comment:
Hmmm, recall havng a discussion on this DeleteDir method. Can we not drop it? It has nothing to do with Browsers.... 

If it is to "Reset" a cahce, it should be on the Cache interface and should be called Rest but I do not think it is wise to allow Javascript user to pass in any directory to be deleted...